<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Add_Logger_Window</class>
 <widget class="QDialog" name="Add_Logger_Window">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>747</width>
    <height>449</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QDialogButtonBox" name="bb_decision">
   <property name="geometry">
    <rect>
     <x>550</x>
     <y>400</y>
     <width>172</width>
     <height>27</height>
    </rect>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
   </property>
  </widget>
  <widget class="QComboBox" name="cb_package">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>180</y>
     <width>341</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="lb_package">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>160</y>
     <width>341</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Package</string>
   </property>
  </widget>
  <widget class="QComboBox" name="cb_message">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>110</y>
     <width>341</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="lb_message">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>90</y>
     <width>341</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Message</string>
   </property>
  </widget>
  <widget class="QLabel" name="lb_type">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>111</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Logger Type</string>
   </property>
  </widget>
  <widget class="QComboBox" name="cb_type">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>40</y>
     <width>161</width>
     <height>31</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>CSV File</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Console</string>
    </property>
   </item>
  </widget>
  <widget class="QLabel" name="lb_file">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>250</y>
     <width>341</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Save file as</string>
   </property>
  </widget>
  <widget class="QTextBrowser" name="id_defs">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>270</y>
     <width>701</width>
     <height>51</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="pb_browse">
   <property name="geometry">
    <rect>
     <x>620</x>
     <y>330</y>
     <width>101</width>
     <height>29</height>
    </rect>
   </property>
   <property name="text">
    <string>Browse</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
